data/schema_data.sql

```diff
@@ -1,188 +1,28 @@
-/*!40000 ALTER TABLE `relatedProducts` ENABLE KEYS */;
-UNLOCK TABLES;
-/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;
-
-/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
-/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
-/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
-/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
-/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
-/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
-/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;
-
--- Dump completed on 2020-05-01 12:52:26
+DROP DATABASE IF EXISTS adidas;
+
+CREATE DATABASE adidas;
+
+USE adidas;
+
+DROP TABLE IF EXISTS categoryInfo;
+CREATE TABLE categoryInfo(
+    categoryId INT AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    categoryName VARCHAR(255)
+) ENGINE=INNODB;
+
+DROP TABLE IF EXISTS productInfo;
+CREATE TABLE productInfo(
+    productId INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
using not null isn't required on a primary key as the system wouldn't allow you to input an empty primary key

Reply:
done